mq/3/on-premise/installation/index.md

```diff
@@ -144,3 +145,104 @@ environment variable `CONFIG_JSON`. An example with mq:
+management. The below instructions could easily be altered for other kinds of
+deployments, but this will be tailored towards a CoreOS cluster. Before you get
+started, See [our requirements](#requirements) for the minimum recommended box configuration.
+The bigger, the better.
+
+We provide a `cloud-config` to make provisioning boxes with a few parameters and
+files pretty easy, you can find it here: [cloud-config](https://github.com/iron-io/enterprise/blob/master/cloud-config).
+WARN: you'll have to enter a new CoreOS discovery url, you can grab one [here](https://discovery.etcd.io/new).
+
+You'll want to at least have the `cloud-config` copied into your clipboard or downloaded, to
+stick it into the platform's configuration. For AWS, you can find this section
+in "Configure instance -> Advanced Details -> user data".
+
+This will not take care of security groups if you're on AWS, so you'll need the
+following ports opened up (note: private ones could be public, but not recommended):
+
+```
+coreos, private ports: 
+:4001 tcp
+:7001 tcp
+
+ironmq, public:
+:8080 tcp
+
+ironmq, private:
+:8081 tcp & udp
+:8082 tcp
+
+ironauth, public:
+:8090 tcp
+
+ironauth, private:
+:8091 tcp & udp
+:8092 tcp
+
+hud-e, public:
+:3000 tcp
+```
+
+You'll probably want an odd numbered amount of boxes, because IronMQ and
+IronAuth both use a quorum-based approach to handle network partitions. We
+recommend starting with 3 nodes and scaling up once you're comfortable with
+managing IronMQ clusters. We also recommend creating a load balancer to point to
+these nodes. IronMQ runs on `:8080` and IronAuth runs on `:8090`. 
+
+After getting the boxes provisioned and running, you can ssh onto one of them to
+take care of fixing up the configs however you might like. They should be in the
+home directory as `mq_config.json` and `auth_config.json`. You should also see
+the service files, we'll need these in a minute.
+
+To get started, you'll have to submit the configs to etcd. For example:
+
+```
+$ etcdctl set /ironmq/config.json “`cat mq_config.json`”
+$ etcdctl set /ironauth/config.json “`cat auth_config.json`”
+```
+
+Afterwards, you can start up the services.
+
+```
+$ fleetctl start ironmq.service
+$ fleetctl start ironauth.service
+$ fleetctl start hud-e.service
+```
+
+You can see that everything is running, and where to find hud-e, by doing:
+
+```
+$ fleetctl list-units
+```
+
+Afterwards, you should be able to go to HUD-e in your browser, it's running on
+port 3000 on the box it landed on. From there, you can use the initial login
+(from `auth_config.json`) to get in, and then we recommend creating other users, 
+tokens and projects to get started. You'll have to add an address in HUD-e in 
+the `manage clusters` section of the admin section for MQ, a load balancer address 
+works great.
+
+This should give you a basic installation of IronMQ, IronAuth and
+HUD-enterprise. We do not recommend using the above instructions in a production
```

Comment:
actually , even then, unless you think mq is more likely to destroy nodes, the availability is the same